src/app/core/ui/search/search.component.ts

```diff
@@ -94,19 +89,10 @@ export class SearchComponent {
     if (this.state !== this.SEARCH_IN_PROGRESS) {
       return [];
     }
-    const searchTerms = next.toLowerCase().split(" ");
-    const entities = await this.indexingService.queryIndexRaw(
-      "search_index/by_name",
-      {
-        startkey: searchTerms[0],
-        endkey: searchTerms[0] + "\ufff0",
-        include_docs: true,
-      }
-    );
-    const filtered = this.prepareResults(entities.rows, searchTerms);
-    const uniques = this.uniquify(filtered);
-    this.state = uniques.length === 0 ? this.NO_RESULTS : this.SHOW_RESULTS;
-    return uniques;
+    const entities = await this.searchService.getSearchResults(next);
+    const filtered = this.prepareResults(entities);
```

Comment:
Should we change MIN_CHARACTERS_FOR_SEARCH to 2 instead of 3 to reasonably support (2 digit) project numbers and the like?